rslint_core/src/groups/errors/no_setter_return.rs

```diff
@@ -0,0 +1,363 @@
+                                if let Some(ast::Expr::ObjectExpr(inner_obj)) = literal_prop.value()
+                                {
+                                    for inner_prop in inner_obj.props() {
+                                        self.check_object_props(
+                                            literal_prop.syntax(),
+                                            &inner_prop,
+                                            ctx,
+                                        );
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            SETTER => {
+                let setter = node.to::<ast::Setter>();
+                if let Some(body) = setter.body() {
+                    if let Some(key) = setter.key() {
+                        self.check_stmts(key.syntax(), body.syntax(), body.stmts(), ctx);
+                    }
+                }
+            }
+            _ => {}
+        }
+        None
+    }
+}
+
+impl NoSetterReturn {
+    fn check_object_props(
+        &self,
+        key: &SyntaxNode,
+        prop: &ast::ObjectProp,
+        ctx: &mut RuleCtx,
+    ) -> Option<()> {
+        match prop {
+            ast::ObjectProp::LiteralProp(literal_prop) => {
+                if literal_prop.key()?.syntax().text() != "set" {
+                    return None;
+                }
+                match literal_prop.value()? {
+                    ast::Expr::FnExpr(decl) => {
+                        self.check_stmts(key, decl.body()?.syntax(), decl.body()?.stmts(), ctx);
+                    }
+                    ast::Expr::ArrowExpr(arrow) => {
+                        if let ast::ExprOrBlock::Block(block) = arrow.body()? {
+                            self.check_stmts(key, block.syntax(), block.stmts(), ctx);
+                        }
+                    }
+                    _ => {}
+                }
+            }
+            ast::ObjectProp::Setter(setter) => {
+                self.check_stmts(key, setter.body()?.syntax(), setter.body()?.stmts(), ctx);
+            }
+            ast::ObjectProp::Method(method) => {
+                if method.name()?.syntax().text() != "set" {
+                    return None;
+                }
+                self.check_stmts(key, method.body()?.syntax(), method.body()?.stmts(), ctx);
+            }
+            _ => {}
+        }
+
+        None
+    }
+
+    fn check_stmts(
+        &self,
+        key: &SyntaxNode,
+        body: &SyntaxNode,
+        mut stmts: impl Iterator<Item = ast::Stmt>,
+        ctx: &mut RuleCtx,
+    ) {
+        if stmts.any(|stmt| self.stmt_returns_value(&stmt)) {
+            let err = ctx.err(
+                self.name(),
+                format!(
+                    "Setter properties are not allowed to return values, but `{}` does.",
```

Comment:
```suggestion
                    "setter properties are not allowed to return values, but `{}` does.",
```
Error messages should be lowercase

Reply:
The error also needs to include a primary label to the error to help the user find out where it is and to be consistent